src/vs/workbench/contrib/markers/common/markersContextKey.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I think this does not need its own file. We can move `MarkersContextKeys` into markers file.

Reply:
Fixed.